app/styles/app.scss

```diff
@@ -818,3 +815,4 @@ span.sortable-bars:hover {
 }
 
 
+
```

Comment:
lol, unless this was intentional, try to be selective about the files you add to your commits!